Library/Formula/gobject-introspection.rb

```diff
@@ -16,7 +16,9 @@ class GobjectIntrospection < Formula
 
   depends_on "pkg-config" => :run
   depends_on "glib"
+  depends_on "cairo"
   depends_on "libffi"
+  depends_on :python
```

Comment:
actually no: there was a bug in at least one of the system python's on the buildbots that causes the gobject-introspection build to fail